save-cloud-charts/save-cloud/templates/demo-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo
+  labels:
+    {{- include "common.labels" (merge (dict "service" .Values.demo) .) | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      io.kompose.service: demo
+  strategy:
+    # Because of shared volume with multi-attach problem
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        {{- include "pod.common.labels" (merge (dict "service" .Values.demo ) .) | nindent 8 }}
+      annotations:
+        {{- include "pod.common.annotations" (dict "service" .Values.demo ) | nindent 8 }}
+    spec:
+      restartPolicy: Always
+      containers:
+        - name: demo
+          {{- include "spring-boot.common" (merge (dict "service" .Values.demo) .) | nindent 10 }}
+          env:
+            {{- include "spring-boot.common.env" (merge (dict "service" .Values.demo) .) | nindent 12 }}
+            - name: JAVA_TOOL_OPTIONS
+              value: -XX:ReservedCodeCacheSize=48M
+          volumeMounts:
+          {{- include "spring-boot.management" .Values.demo | nindent 10 }}
```

Comment:
This thing should be on the same level as `volumeMounts` (and not part of it). You can see how it's expanded in `_helpers.tpl`

Reply:
Fixed it